src/components/button-load.js

```diff
@@ -0,0 +1,7 @@
+const createButtonLoadMoreTemplate = () => {
```

Comment:
не соответствует названию файла
```suggestion
const createButtonLoadTemplate = () => {
```
либо переименуй файл